dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1490,6 +1490,24 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 		return;
 	}
 
+	//TESTING IMMUNITY
+	if (chatCommand == "immune" && args.size() >= 1 && entity->GetGMLevel() >= eGameMasterLevel::DEVELOPER) {
```

Comment:
If this is going to be merged as well, the name of the command should be a bit more specific as to what you are immune to (in this case basic attack immunity) and should be documented in the Docs, or should be removed, up to you which path you take